<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwedigarchExportDialogBase</class>
 <widget class="QDialog" name="SwedigarchExportDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intrasis DB Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/svedigark.svg</normaloff>assets/svedigark.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_db_conect">
       <item>
        <widget class="QLabel" name="label_Server">
         <property name="text">
          <string>Database connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_DbConnection">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>22</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pgSelectConnection">
         <property name="toolTip">
          <string>Select stored PostgreSQL connection (stored by DB Manager)</string>
         </property>
         <property name="text">
          <string>Select connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbConnect">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Connect...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbDisconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSearch">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>61</height>
        </size>
       </property>
       <property name="title">
        <string>Filter databases</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_search">
          <item>
           <widget class="QLineEdit" name="lineEditSearch">
            <property name="maximumSize">
             <size>
              <width>377</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblSearchInfo">
            <property name="text">
             <string>search info text</string>
            </property>
            <property name="margin">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxDatabases">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Doubleclick to move databases between the lists</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="title">
        <string>Available databases</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="lwDatabases">
            <property name="toolTip">
             <string>Doubleclick to move databaases between lists</string>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pbSelectAllDb">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Add / remove all databases</string>
              </property>
              <property name="text">
               <string>Select all databases</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>12</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbAdd">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Add selected databases</string>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>assets/arrow_down.png</normaloff>assets/arrow_down.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>12</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbRemove">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Remove selected databases</string>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>assets/arrow_up.png</normaloff>assets/arrow_up.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSelected">
       <property name="toolTip">
        <string>Doubleclick to move databaases between lists</string>
       </property>
       <property name="title">
        <string>Selected databases for export</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QListWidget" name="lwSelectedDatabases">
          <property name="toolTip">
           <string>Doubleclick to move databaases between lists</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Export catalog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditExportDirectory"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowse">
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="cbOverwriteExistingGeoPackage">
         <property name="toolTip">
          <string>If the GeoPackage already exist it will be overwritten if checked.
Otherwice the GeoPackage will not be exported.</string>
         </property>
         <property name="text">
          <string>Overwrite existing GeoPackage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbExportCSV">
         <property name="toolTip">
          <string>Should an CSV export also be done for every exported database</string>
         </property>
         <property name="text">
          <string>Export CSV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbFilterSubClass">
         <property name="text">
          <string>Filter by SubClass</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSimplifiedExport">
         <property name="toolTip">
          <string>Should an CSV export also be done for every exported database</string>
         </property>
         <property name="text">
          <string>Simplified Export</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="help_button">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>SwedigarchExportDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>SwedigarchExportDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
